### PR TITLE
Fix Arabic words gender

### PR DESCRIPTION
**Pull request by elkebirmed**
## Before:
الإسم مطلوب :heavy_check_mark: 
كلمة المرور مطلوب :x: 

## Now:
حقل الإسم مطلوب :heavy_check_mark: 
حقل كلمة المرور مطلوب :heavy_check_mark:

### CODE REVIEW

File: locales/ar/validation.php
Comment (caouecs) [inline locales/ar/validation.php:15]: Don't update spaces, please

**Review by caouecs — CHANGES_REQUESTED** [review]
Problem with indentation

**Review by mohamedsabil83 — CHANGES_REQUESTED** [review]
We can approve it after fixing the indentation

**Review by mohamedsabil83 — APPROVED** [review]
@elkebirmed Thank you.
@caouecs You can accept it now.

**Comment by mohamedsabil83** [comment]
@elkebirmed Please fix the conflict or you can **Allow edits from maintainers** to this pr.

**Comment by elkebirmed** [comment]
@mohamedsabil83 @caouecs Conflict resolved :+1:

**Comment by mohamedsabil83** [comment]
You can merge it now @caouecs

**Comment by caouecs** [comment]
Thank you

**Comment by elkebirmed** [comment]
@caouecs You are welcome

**Pull request merged**
